official/utils/logs/logger.py

```diff
@@ -39,6 +40,10 @@
 BENCHMARK_RUN_LOG_FILE_NAME = "benchmark_run.log"
 _DATE_TIME_FORMAT_PATTERN = "%Y-%m-%dT%H:%M:%S.%fZ"
 
+GCP_TEST_ENV = "GCP"
+GCP_METADATA_URL = "http://metadata/computeMetadata/v1/instance/hostname"
+GCP_METADATA_HEADER = {"Metadata-Flavor": "Google"}
```

Comment:
Might be wise to start separating out some of the system-specific logic. For example, having a gcp_utils in this dir that has a function that is is_gcp() or something like that. Otherwise, AWS/etc. will feel "tacked on" and it will get messy.

Reply:
Done. extract to another lib.